src/lib/Components/Bidding/Elements/Flex.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components/native"
+import { borders, color, flex, fontSize, space, width } from "styled-system"
+
+/* FIXME: Do not use any */
+export const Flex: any = styled.View`
```

Comment:
I forgot to mention in the first review, but this `any` stinks very bad... I'll look into this deeper but this may be something we may have to contribute back to `styled-system` or `DefinitelyTyped` to get the the benefit of the type system.